community/wireguard-go/APKBUILD

```diff
@@ -0,0 +1,26 @@
+# Contributor: Stefan Reiff <kroko87@hotmail.com>
+# Maintainer: Stefan Reiff <kroko87@hotmail.com>
+pkgname=wireguard-go
+pkgver=0.0.20191012
+pkgrel=0
+pkgdesc="Next generation secure network tunnel: userspace implementation in go"
+arch="all"
+url="https://www.wireguard.com"
+# SPDX identifier headers tells us 'GPL-2.0' but considering it
+# is a kernel project i think it is safe to assume it is GPL-2.0-only just
+# like the kernel.
```

Comment:
You could just grep the source for "at your option" since that's part of GPL-2.0-or-later

Reply:
Couldn't find such string, is "custom" ok for this package?

Reply:
If there's no such string then it's GPL-2.0-only :)

Reply:
Ah ok ^^. Thanks!